cluster/saltbase/salt/kubelet/default

```diff
@@ -102,5 +102,10 @@
   {% set test_args=pillar['kubelet_test_args'] %}
 {% endif -%}
 
+{% set privileged = pillar.get('allow_privileged', false)  -%}
+{% if pillar.get('e2e_test_environment', '').lower() == 'true' -%}
+  {% set privileged = "True" -%}
```

Comment:
I think we're about to get into a divide on "a set of tests that can run on more or less vanilla clusters" and "stuff that needs invasive permissions". Can you really not do this without such invasive changes? I'm not totally against the package changes (maybe) but the environmental changes are going to block this for now.